lib/containers/runtime.pm

```diff
@@ -131,18 +214,85 @@ sub remove_image {
     $self->_rt_assert_script_run("rmi -f $image_name");
 }
 
+=head2 remove_container
+
+Remove a container from the pool.
+
+=cut
+sub remove_container {
+    my ($self, $container_name) = @_;
+    $self->_rt_assert_script_run("rm -f $container_name");
+}
+
+=head2 confirm_image_in_host_registry
```

Comment:
I find `confirm` the wrong wording here and would use something like `has_image` or `check_image_in_host_registry`.
IMHO confirm goes more in the direction of an assert. With that name, I would assume the function checks if the image is in the host registry and fails otherwise. Confirm suggests an action not a check.

Sorry for the nitpicking :slightly_smiling_face: